packages/grpc-js/src/subchannel.ts

```diff
@@ -227,6 +227,8 @@ export class Subchannel {
     }
     const previousState = this.connectivityState;
     this.connectivityState = newState;
+    process.nextTick(() => {
```

Comment:
What's the purpose of this call?
It looks like some debugging leftover. It doesn't make anything asynchronous, it just adds an empty function to be called by Event Loop after synchronous part of the code is executed.

Reply:
You're right. That was part of a change that I was trying out before I went in a different direction.